Add milvus example mistral french parliament

stephen37 opened the pull request:
This is an example of how one can build a RAG app and interact with data from the French parliament using Milvus as the Vector DB an Mistral as the LLM in combination with Ollama.

sophiamyang commented:
Hi @stephen37 thanks for contributing! could you add the notebook to the third-party integrations folder and could you use Mistral embeddings? Thanks!

pandora-s-git commented:
Another option since the issue seems to be the fact that the data is originally French would be to switch to a different one in English.

stephen37 commented:
> Hi @stephen37 thanks for contributing! could you add the notebook to the third-party integrations folder and could you use Mistral embeddings? Thanks!

Hey @sophiamyang, I moved it to the third party folder and changed to Mistral Embeddings, let me know what you think now 🙏

sophiamyang commented:
Hi @stephen37 , thank you so much for the cookbook! We are ready to merge this cookbook! However, I got an error:
```ValueError: ServiceContext is deprecated. Use llama_index.settings.Settings instead,``` 

Could you help update the syntax and tag the package version in case llamaindex has more changes? Also, wondering if you could help add another example using Mistral API in addition to Ollama? Thanks so much!

stephen37 commented:
@sophiamyang Sorry for the delay, I was on holiday last week but I pushed a new commit addressing your concerns, let me know what you think :)

sophiamyang commented:
Hi @stephen37 , thanks so much for the updates!! Looks good. However, I got an error message: "ValueError: File data/french_parliament_discussion.xml does not exist.". Could you help fix? Thanks!

ravi03071991 commented:
> Hi @stephen37 , thanks so much for the updates!! Looks good. However, I got an error message: "ValueError: File data/french_parliament_discussion.xml does not exist.". Could you help fix? Thanks!

fixed this.
